geth/rpc/client_test.go

```diff
@@ -46,7 +46,7 @@ func (s *RPCTestSuite) SetupTest() {
 
 func (s *RPCTestSuite) TestRPCSendTransaction() {
 	require := s.Require()
-	expectedResponse := []byte(`{"jsonrpc": "2.0", "status":200, "result": "3434=done"}`)
+	expectedResponse := []byte(`{"jsonrpc":"2.0","id":10,"result":"3434=done"}`)
```

Comment:
Have you talked to status-react about this?

Reply:
@tiabc I tracked down git history, and I'm pretty sure it's mistake in a test: the code returning `"status":200` is only mocked `httptest.Server` handler (introduced in the same commit). JSON-RPC standard doesn't contain `Status` field either. I also more or less familriazed with `web3.js` responsible for JSON-RPC and there is no usage of such field as well.

Reply:
Anyway, please, consult status-react about this. Just to be sure we're not missing anything.

Reply:
Done